### PR TITLE
update

### PR DESCRIPTION
**Pull request by melnicenkovadik**

### CODE REVIEW

File: src/pages/Staking/Staking.js [hunk 1]
```diff
@@ -51,21 +51,26 @@ const Staking = observer(() => {
...
-                comingSoon={!item?.abi}
-                lazy
-                poolInfo={item}
-              />
-            ))}
+            {pools
+              .sort(function (a, b) {
+                return a.active - b.active;
+              })
+              .reverse()
```
Comment (vitalii427) [inline src/pages/Staking/Staking.js:58]: Why revese? Make correct sorting.

File: src/Main.js [hunk 2]
```diff
@@ -18,8 +18,12 @@ const Main = observer(() => {
       appStore.setAuth(true);
     }
     return () => {
-      ethereum.on('accountsChanged', () => null);
-      ethereum.on('chainChanged', () => null);
+      if (ethereum.removeListener) {
+        ethereum.removeListener('chainChanged', () => window.location.reload());
```
Comment (vitalii427) [inline src/Main.js:22]: This is wrong! You should remove previously defined function, not the new lambda.

File: src/Main.js [hunk 3]
```diff
@@ -18,8 +18,12 @@ const Main = observer(() => {
       appStore.setAuth(true);
     }
     return () => {
-      ethereum.on('accountsChanged', () => null);
-      ethereum.on('chainChanged', () => null);
+      if (ethereum.removeListener) {
```
Comment (vitalii427) [inline src/Main.js:21]: What is this check for? And what if there's no ethereum.removeListener ?

File: src/Main.js (hunk 2 above)
Comment (melnicenkovadik) [reply src/Main.js:22]: example here => https://github.com/NoahZinsmeister/web3-react/blob/v6/example/hooks.ts

File: src/pages/Staking/Staking.js (hunk 1 above)
Comment (melnicenkovadik) [reply src/pages/Staking/Staking.js]: +

**Pull request closed without merging**